lib/menu.rb

```diff
@@ -0,0 +1,29 @@
+require 'csv'
+
+class Menu
```

Comment:
This seems mainly a file reading class. I'd consider intializing it with the file path and wrapping some of the reading in methods to reduce the amount that's publicly exposed if that makes sense? maybe at some point creating a csv reading class. 

Essentially, off the top of my head, i'd consider something like this:

```ruby
def initialize
  @menu = []
end

def menu
  create_menu_from_csv
  @menu
end

private

def create_menu_from_csv
  CSV.foreach("menu.csv") do |row|
  name, price = row
  @menu << { name: name, price: price.to_i }
  end
end
```